test/controllers/creditors_controller_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+class CreditorsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I wonder why these tests have been created with Minitest instead of RSpec, maybe we missed something when setting up RSpec? 

For now let's just delete the whole `test` folder, then we create the RSpec tests manually later